packages/mirrorful/editor/src/blocks.css

```diff
@@ -1,4 +1,4 @@
-@import '../.mirrorful/theme.css';
+@import '../../.mirrorful/theme.css';
```

Comment:
@Teddarific just fyi on this. I'm a bit confused here because there's no .mirrorful in source. so not sure what it's even using. Delegating to you because I know you were setting up blocks.css